pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -83,8 +83,9 @@ const (
 	// ReasonInvalidWorkspaceBinding indicates that a Pipeline expects a workspace but a
 	// PipelineRun has provided an invalid binding.
 	ReasonInvalidWorkspaceBinding = "InvalidWorkspaceBindings"
-	// ReasonInvalidServiceAccountMapping indicates that PipelineRun.Spec.TaskRunSpecs[].TaskServiceAccountName is defined with a wrong taskName
-	ReasonInvalidServiceAccountMapping = "InvalidServiceAccountMappings"
```

Comment:
🤔 Couldn't this happen still ?
I understand that we were sending this error wrongly (as it could have bee n something different that invalid service account mapping, but.. could/would we have cases where issuing this error is still valid ?

Reply:
ok, i kept this reason.

Reply:
I think this reason related function is removed, and deprecated? I don't see a reason to keep it.
https://github.com/tektoncd/pipeline/pull/3028

Reply:
It is indeed not used here. This commit will be deleted first, and if it is used again later, it will be added?